doc/api/number/number-formatter.md

```diff
@@ -94,7 +100,9 @@ Globalize.numberFormatter({ minimumFractionDigits: 2 })( 1.5 );
 // > "1.50"
 ```
 
-Controlling digits by specifying significant digits counts:
+#### Configuring significant decimal places
```

Comment:
This seems wrong, this isn't about "places", but about "digits", though I don't understand what `minimumSignificantDigits` and `maximumSignificantDigits` actually do. The examples are not helping.